### PR TITLE
Add GitHub release docs

### PR DESCRIPTION
**Pull request by EMaksy**
Add a guide how to release Trento on GitHub.

### CODE REVIEW

File: guides/release/github-release.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+### Step Three: Version Bump
+
+1. **Version Upgrade**
+
+   - Search for the current version in every project file and perform a version bump.
+
+   Previous Release Examples:
+
+   - [Agent](https://github.com/EMaksy/agent/commit/df9bce2692ee46d3faa548494ec7ba40a22d1873)
```
[the remark this answers is not in the document]
Comment (EMaksy) [reply guides/release/github-release.md:61]: Oh thanks for the catch! :)

File: guides/release/github-release.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
+# GitHub Release Process
+
+## Step Zero: Requirements:
```
[the remark this answers is not in the document]
Comment (EMaksy) [reply guides/release/github-release.md]: Good idea, looks better now :)

File: guides/release/github_release/github-release.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+
+### Step Three: Version Bump
+
+1. **Version Upgrade:**
+
+   - Search for the current version in the project and perform a version bump.
+
+   Examples:
+
+   - [Agent](https://github.com/EMaksy/agent/commit/df9bce2692ee46d3faa548494ec7ba40a22d1873)
```
[the remark this answers is not in the document]
Comment (EMaksy) [reply guides/release/github_release/github-release.md]: YES!
A leftover from playing around before :)

**Review by nelsonkopliku — APPROVED** [review]
Thanks!

File: guides/release/community-release.md [hunk 4]
```diff
@@ -0,0 +1,98 @@
+# GitHub Release Process
+
+## 1. Requirements
+
+a. **GitHub Token**
```
Comment (arbulu89) [inline guides/release/community-release.md:5]: I still continue finding the indentation too much. 
I would remove the letter based bullet point or the next bullet points.
All the entries only have one unique entry...
```
1.
b.
-
```
XD

File: guides/release/community-release.md [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+
+### 4. Version Bump
+
+a. **Version Upgrade**
+
+- Search for the current version in every project file and perform a version bump.
+
+Previous Release Examples:
+
+- [Agent](https://github.com/trento-project/agent/commit/df9bce2692ee46d3faa548494ec7ba40a22d1873)
```
Comment (arbulu89) [inline guides/release/community-release.md:69]: I still think that a link to a release would be better, as some release PRs might have multiple commits

File: guides/release/community-release.md [hunk 6]
```diff
@@ -0,0 +1,98 @@
+# GitHub Release Process
+
+## 1. Requirements
```
Comment (arbulu89) [inline guides/release/community-release.md:3]: Is there any reason why this header is of `##` level, and the rest of 3 `###`?

**Review by arbulu89 — APPROVED** [review]
Great!

File: guides/release/community-release.md (hunk 6 above)
Comment (EMaksy) [reply guides/release/community-release.md:3]: a very good question :) 
I will remove it!

File: guides/release/community-release.md (hunk 4 above)
Comment (EMaksy) [reply guides/release/community-release.md:5]: Fair point, i will remove one level: '-'

Comment (EMaksy) [reply guides/release/community-release.md]: and i removed the letters, same style as on the [hack on trento guide](https://github.com/trento-project/web/blob/main/guides/development/hack_on_the_trento.md)

**Pull request merged**